packages/context/test/unit/resolver.test.ts

```diff
@@ -257,7 +257,7 @@ describe('constructor injection', () => {
         (c: Context, injection: Injection, session: ResolutionSession) => {
           bindingPath = session.getBindingPath();
           resolutionPath = session.getResolutionPath();
-          decorators = session.injectionStack.map(i => i.metadata!.decorator);
+          decorators = session.injectionStack.map(i => i.metadata!.decorator!);
```

Comment:
I am not very happy about shutting up compiler's complaint about possibly-null `decorator` value. Have you considered changing the type of `decorators` to `(string | undefined)[]` instead?